COMPILE.md

```diff
@@ -0,0 +1,13 @@
+## Compilation Instructions
```

Comment:
* [ ] Merge with `README.md`
* [ ] Add testing instructions

Reply:
The problem is that the README is what gets rendered as the text in the marketplace and in the extension's description.
That's the reason it is a separate file right now

Reply:
OK, then this file should be called `INSTALL.md`.

Reply:
Fixed